config/packages/security.yaml

```diff
@@ -54,6 +54,7 @@ security:
 
     access_control:
          - { path: ^/admin, roles: ROLE_ADMIN }
+         - { path: ^/user/invitation., roles: [ROLE_USER, IS_AUTHENTICATED_ANONYMOUSLY] }
```

Comment:
this `.` is for a purpose?

Reply:
I've just cp your suggestion from https://github.com/repman-io/repman/pull/387#discussion_r566313300
Should I remove it?
Also PR rebased on master

Reply:
Ok, it's my mistake then :wink:, please remove it, and we can merge this :+1:

Reply:
and sorry for the trouble, I wrote it from memory